Modules/Core/Common/include/itkBSplineDerivativeKernelFunction.h

```diff
@@ -168,8 +168,6 @@ class BSplineDerivativeKernelFunction:public KernelFunctionBase<TRealValueType>
   inline TRealValueType Evaluate( const DispatchBase&, const TRealValueType& ) const
     {
     itkExceptionMacro( "Evaluate not implemented for spline order " << SplineOrder );
-    return NumericTraits< TRealValueType >::ZeroValue(); // This is to avoid compiler warning about missing
```

Comment:
I guess compilers have matured since this line was written, and they now complain about presence of this rather than absence of it 😄